humble/distribution.yaml

```diff
@@ -1938,6 +1938,26 @@ repositories:
       url: https://github.com/dynamixel-community/dynamixel_hardware.git
       version: humble
     status: developed
+  dynamixel_hardware_interface:
+    doc:
+      type: git
+      url: https://github.com/ROBOTIS-GIT/dynamixel_hardware_interface.git
```

Comment:
In previous versions of ROS 2, this was released from the [OUXT-Polaris repository](https://github.com/ros/rosdistro/blob/411217ce18e43e6023cdad68ef62d39bb2b32845/foxy/distribution.yaml#L1131) by @hakuturu583 .  What's the relationship between this repository and that one?

Reply:
We didn't realize there was a package with the same name. 
@hakuturu583, is this package going to be continuously developed in the future?
If so, we will rename it to something else, please confirm it.

Reply:
@hakuturu583, We'll rename the package, and we sincerely apologize for the confusion.
@clalancette, We'll come back to this after we've renamed the package and made a revision commit with the relevant changes.

Reply:
Thank you for notification.
That package is the one we developed when we configured azimuth thrusters for small autonomous vessels with Dynamixel.

https://forum.robotis.com/t/mini-v/1793

We developed and released it because there was no official hardware_interface at that time, but we do not intend to release the package in the future because OUXT Polaris does not need to release it as long as the official release is available.

Reply:
@hakuturu583 
We appreciate your contributions and sincerely apologize for any disappointment in customer support that we may have caused by waiting too long to do what we should have done. We sincerely appreciate your contributions.
So, can we use the name dynamixel_hardware_interface starting with ROS 2 humble?

Reply:
@clalancette As mentioned above, the previous package manager (@hakuturu583) is no longer scheduled for release. So is it okay for our team to use this name (dynamixel_hardware_interface) starting with the humble release?